svf/lib/AE/Svfexe/AbsExtAPI.cpp

```diff
@@ -340,6 +340,18 @@ void AbsExtAPI::initExtFunMap()
     };
     func_map["recv"] = sse_recv;
     func_map["__recv"] = sse_recv;
+     
+     auto sse_free = [&](const CallICFGNode *callNode)
+     {
+         if (callNode->arg_size() < 1) return;
+         AbstractState& as = getAbsStateFromTrace(callNode);
+         const u32_t freePtr = callNode->getArgument(0)->getId();
+         for (auto addr: as[freePtr].getAddrs()) {
+             NodeID addrId = AbstractState::getInternalID(addr);
+             as[addrId] = AddressValue(DanglingPtrVal);
+         }
+     };
+     func_map["free"] = sse_free;
```

Comment:
not only "free" functions. Should use all "CK_FREE" functions. We could move the below functions to extapi.c

https://github.com/SVF-tools/SVF/blob/35217034ccd43df212b5a18a8855cc1b35459d45/svf/lib/SABER/SaberCheckerAPI.cpp#L51-L136